scout/server/blueprints/variant/utils.py

```diff
@@ -3,6 +3,7 @@
 
 from scout.adapter import MongoAdapter
 from scout.constants import ACMG_COMPLETE_MAP, CALLERS, CLINSIG_MAP, SO_TERMS
+from scout.server.extensions import store
```

Comment:
I would suggest passing store as an argument. It is a bit arbitrary. Parse is the most important to keep clean. This particular file will not be involved in an import cycle any day soon. But since the design is different from essentially all other controllers and their util functions: is there any particular reason to import a global module variable rather than pass the instance of it?